pydata/pydata-sphinx-theme
# BUG - center search dialog on wide screens

I noticed while reading the docs that on very wide screens, the search dialog was off center. 

This PR gets rid of all unnecessary CSS that was left over from when the search dialog was implemented without the built-in benefits of the HTML modal dialog element.

### How to test

1. Go to the preview docs build for this PR.
2. Make your screen really wide > 2000px.
3. Click on the search button in the header nav.
4. The search dialog should be centered now.
5. For good measure, type something in the search bar and hit enter.
